request-manager/template/grapher.go

```diff
@@ -228,31 +230,34 @@ func (o *Grapher) getTemplate(seq *spec.Sequence) (*Graph, error) {
 	jobArgs := getAllSequenceArgs(seq)
 
 	// Create a graph node for every node in the spec
-	components := map[*spec.Node]*Node{}
+	// Key on node names--they should be unique within a sequence (otherwise, dependencies
```

Comment:
err, is there a check that node names are unique within a sequence?

Reply:
The yaml parser errors if that happens. I don't remember the exact error message, but I think it was sufficiently descriptive? Either way, I seem to remember that there's no good way to distinguish between different types of errors thrown by the parser.

Reply:
Ah neat, that works.